### PR TITLE
Fix order of paginated accounts in FollowerDomainsController and spec

### PR DESCRIPTION
**Pull request by akihikodaki**
Unordered pagination could result in unexpected behavior.

### CODE REVIEW

File: spec/controllers/settings/follower_domains_controller_spec.rb [hunk 1]
```diff
@@ -5,32 +5,81 @@
...
+    subject { get :show, params: { page: 2 } }
+
+    it 'assigns @account' do
+      sign_in user, scope: :user
+      subject
+      expect(assigns(:account)).to eq user.account
+    end
+
+    it 'assigns @domains' do
+      followers = 11.times.map do |index|
```
Comment (mjankowski) [inline spec/controllers/settings/follower_domains_controller_spec.rb]: Override pagination threshold instead of creating so many records?

**Review by mjankowski — DISMISSED** [review]

File: spec/controllers/settings/follower_domains_controller_spec.rb (hunk 1 above)
Comment (akihikodaki) [inline spec/controllers/settings/follower_domains_controller_spec.rb]: It is not good for testing the query... I'm still considering other means, but that is possible as the last resort.

Comment (mjankowski) [inline spec/controllers/settings/follower_domains_controller_spec.rb]: Can we move the huge string of methods in the controller into some scopes and class methods instead? That might make it easier to have a more thorough spec for the query improvement (in a model spec) while keeping the controller spec more focused, and the controller code more readable.

Comment (akihikodaki) [inline spec/controllers/settings/follower_domains_controller_spec.rb]: I'm :+1: for the idea, but I'd like to focus on the fix of the order and spec of the current implementation in this pull request.

**Comment by akihikodaki** [comment]
0b21068efe529eb195a9ac925a4e0cca28932438 overrides `per` as suggested.

**Comment by unarist** [comment]
@mjankowski Have you checked update after your review?

**Review by Gargron — APPROVED** [review]

**Pull request merged**